packages/@webex/plugin-cc/src/services/core/WebSocket/keepalive.worker.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+// TODO: Try to find alternative to using Blob and script here
+const workerScript = `
```

Comment:
As mentioned, this is a workaround, currently we export the template string and make it a Blob and use it as a worker script.
Work is being done to explore a better method for this, however to unblock we can go ahead with this workaround.
Open to any suggestions.